modules/virt-querying-metrics.adoc

```diff
@@ -66,12 +66,24 @@ Returns the total amount (in bytes) of the virtual machine's storage-related tra
 `kubevirt_vmi_storage_write_traffic_bytes_total`::
 Returns the total amount of storage writes (in bytes) of the virtual machine's storage-related traffic.
 
-.Example storage-related traffic query
+`kubevirt_vmsnapshot_disks_restored_from_source_total`::
+Returns the total number of virtual machine disks restored from the source virtual machine.
+
+`kubevirt_vmsnapshot_disks_restored_from_source_bytes`::
+Returns the amount of space in bytes restored from the source virtual machine.
+
+.Examples of storage-related traffic queries
```

Comment:
The [OCP docs contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) state "Do not use more than one command per code block." I think that still applies here, so please split this up into three blocks.

Reply:
Yep, sounds good.